src/main.rs

```diff
@@ -23,11 +26,19 @@ fn main() {
     let client = reactor.prepare_client_and_connect(&config).unwrap();
     client.identify().unwrap();
 
+    let mut ratelimit = ratelimit::Builder::new()
+        .capacity(10) //number of tokens the bucket will hold
+        .quantum(5) //add five tokens per interval
+        .interval(Duration::new(30, 0)) //add quantum tokens every 30 seconds
```

Comment:
smol comment nit, "add quantum tokens" initially read as "add <quantum tokens>", not "add <quantum> tokens". backticks would probably help or something but 🤷‍♀️